bundle/Resources/views/Limitation/udw_limitation_value.html.twig

```diff
@@ -0,0 +1,14 @@
+{{ form_label(form.limitationValues) }}
+{{ form_errors(form.limitationValues) }}
+{{ form_widget(form.limitationValues) }}
+
+<button data-universaldiscovery-title="{{"role.policy.limitation.location.udw_title"|trans({}, "ezrepoforms_role")}}"
+        class="ez-button-tree pure-button ez-font-icon ez-button ez-pick-location-limitation-button"
+        data-location-input-selector="#{{form.limitationValues.vars.id}}"
+        data-selected-location-selector="#{{form.limitationValues.vars.id}}-selected-location"
+>{{"role.policy.limitation.location.udw_button"|trans({}, "ezrepoforms_role")}}</button>
+<div id="{{form.limitationValues.vars.id}}-selected-location">
+    {% if form.limitationValues.vars.value is not empty %}
+        {{ render( controller( "ez_content:viewLocation", {'locationId': form.limitationValues.vars.value, 'viewType': '_platformui_content_name'} ) ) }}
```

Comment:
Deprecation warning!

You should now use `ez_content:view`, passing `contentId` instead of `locationId`. `viewType` is still valid.